move_shows/src/main.py

```diff
@@ -124,8 +128,29 @@ def main():
         )
         if source_show.show_id == dest_show.show_id:
             raise Exception("Source and destination show cannot be the same")
-
-        source_sequences = get_sequences(cur, source_show.show_id)
+        if source_show.episodic != dest_show.episodic:
+            raise Exception(
+                "Source and destination shows must be either both episodic or both non-episodic"
+            )
+
+        source_episode_id: Optional[int] = None
+        dest_episode_id: Optional[int] = None
+        if source_show.episodic == 1:
```

Comment:
```suggestion
        if source_show.episodic:
```
Marginally more robust (and just good practice unless you specifically only want to support certain types), will execute if `episodic` is 1 or `True`, and won't execute if it's 0, `False` or `None`